app/controllers/api/projects.php

```diff
@@ -119,6 +119,11 @@
         if ($index !== false) {
             $dsn = $databases[$index];
         } else {
+
+            $databases = array_filter($databases, function($value, $region) {
+                return str_contains($value, $region);
+            });
```

Comment:
this will be a problem on self hosted, because the region there is `default` and I dont think the database connection string in self hosted has `default` in its name?